cdap-client-tests/src/test/java/co/cask/cdap/client/PreferencesClientTestRun.java

```diff
@@ -48,10 +62,64 @@ public void setUp() throws Throwable {
     super.setUp();
     client = new PreferencesClient(clientConfig);
     appClient = new ApplicationClient(clientConfig);
+    serviceClient = new ServiceClient(clientConfig);
+    programClient = new ProgramClient(clientConfig);
     namespaceClient = new NamespaceClient(clientConfig);
   }
 
   @Test
+  public void testProgramAPI() throws Exception {
```

Comment:
We would need to add namespace-related logic here later right? Could you please add a comment?